lib/pgerd/erd.rb

```diff
@@ -82,7 +93,16 @@ def draw_table(table)
     end
 
     def draw_the_foreign_keys
-      @database.foreign_keys.each { |fk| draw_foreign_key(fk) }
+      progress = ProgressBar.create({
+        :title => 'Foreign Keys',
+        :total => @database.foreign_keys.count,
+        :format => '%t: |%B| %%%p %E',
+	:output => $stderr
+      })
```

Comment:
```suggestion
      progress = ProgressBar.create(
        title: 'Foreign Keys',
        total: @database.foreign_keys.count,
        format: '%t: |%B| %%%p %E',
	output: $stderr)
```

Reply:
```
Carls-MacBook-Pro:pg-erd Carl$ system_profiler SPHardwareDataType
Hardware:

    Hardware Overview:

      Model Name: MacBook Pro
      Model Identifier: MacBookPro12,1
      Processor Name: Dual-Core Intel Core i5
      Processor Speed: 2.7 GHz
      Number of Processors: 1
      Total Number of Cores: 2
      L2 Cache (per Core): 256 KB
      L3 Cache: 3 MB
      Hyper-Threading Technology: Enabled
      Memory: 8 GB
      System Firmware Version: 428.60.3.0.0
      OS Loader Version: 540.80.2~11
      SMC Version (system): 2.28f7
      Serial Number (system): C02P6CP7FVH3
      Hardware UUID: 6E7A5D5F-AE14-55C5-A71E-7499EE29963F
      Provisioning UDID: 6E7A5D5F-AE14-55C5-A71E-7499EE29963F

Carls-MacBook-Pro:pg-erd Carl$ git checkout master
Already on 'master'
Your branch is up to date with 'origin/master'.
Carls-MacBook-Pro:pg-erd Carl$ time bin/pg-erd vanilla-ice > erd.dot

real	3m46.215s
user	0m1.538s
sys	0m0.226s
Carls-MacBook-Pro:pg-erd Carl$ git checkout progressbar
Switched to branch 'progressbar'
Your branch is up to date with 'origin/progressbar'.
Carls-MacBook-Pro:pg-erd Carl$ time bin/pg-erd vanilla-ice > erd.dot
Tables: |===========================================================================================================================================================================================| %100 Time: 00:00:00
Foreign Keys: |=====================================================================================================================================================================================| %100 Time: 00:00:00

real	0m3.546s
user	0m1.359s
sys	0m0.202s
```

Reply:
I'm going to merge my suggestion here, I guess you're okay with that since you merged the previous one that was very similar; and then I'll merge the whole PR.